msal/tests/dev apps/DesktopTestApp/MainForm.cs

```diff
@@ -158,6 +161,45 @@ private async void AcquireTokenInteractive_Click(object sender, EventArgs e)
             }
         }
 
+        private async void acquireTokenByWindowsIntegratedAuth_Click(object sender, EventArgs e)
+        {
+            ClearResultPageInfo();
+
+            string username = loginHintTextBox.Text; // Can be blank 
+
+            try
+            {
+                var app = new PublicClientApplication(publicClientId, authority.Text);
+                AuthenticationResult authenticationResult = await app.AcquireTokenByIntegratedWindowsAuthAsync(scopes.Text.AsArray(), username);
+                SetResultPageInfo(authenticationResult);
+                
+            }
+            catch (Exception exc)
+            {
+                CreateException(exc);
+            }
+        }
+               
+        private async void acquireTokenByUPButton_Click(object sender, EventArgs e)
+        {
+            ClearResultPageInfo();
+
+            string username = loginHintTextBox.Text; //Can be blank for U/P
+            var password = Microsoft.VisualBasic.Interaction.InputBox("Password?");
```

Comment:
> Microsoft.VisualBasic.Interaction.InputBox("Password?"); [](start = 26, length = 57)

It is possible to set the text box in the ui to be a password box... then no need to introduce a VB control?